README.md

```diff
@@ -241,6 +241,7 @@ n1.2xlarge  |  16  |  32Gi
 n1.4xlarge  |  32  |  64Gi
 n1.8xlarge  |  64  |  128Gi
 n1.large  |  4  |  8Gi
+n1.medium  |  4  |  4Gi
```

Comment:
nit: Please consider moving it before `n1.large`.

Reply:
The list is auto-generated and sorted alphabetically. :)